src/Tribe/Views/V2/View.php

```diff
@@ -1039,9 +1046,25 @@ protected function setup_template_vars() {
 				'month_and_year' => tribe_get_date_option( 'monthAndYearFormat', 'F Y' ),
 			],
 			'messages'          => $this->get_messages( $events ),
+			'breadcrumb_header' => [],
 			'start_of_week'     => get_option( 'start_of_week', 0 ),
 		];
 
+		/**
+		 * @todo  @bordoni move this to a template vars modifier of sorts, pending convo with @lucatume
+		 */
+		if ( ! empty( $this->context->get( 'taxonomy' ) ) && ! empty( $this->context->get( $this->context->get( 'taxonomy' ) ) ) ) {
```

Comment:
Please, move the calls to `$this->context->get('taxonomy')` to a var and use that, e.g. `$tax` to make the code more readable.

Reply:
On the same note:

```php
$tax = $this->context->get( 'taxonomy', false );
$term_slug = $tax ? $this->context->get( $tax, false );

if ( $term_slug ){
    $term = get_term_by( 'slug', $term_slug ), $tax );

    $template_vars['breadcrumb_header'][] = [
					'priority' => 10,
					'slug'     => 'taxonomy-' . $tax,
					'html'     => $term->name,
				];
}
```